api/room/schema_query.py

```diff
@@ -368,12 +371,17 @@ def resolve_analytics_ratios(
         return ratio
 
     @Auth.user_roles('Admin', 'Default User')
-    def resolve_analytics_ratios_per_room(
-            self, info, start_date, end_date=None):
+    def resolve_analytics_ratios_per_room(self, info, **kwargs):
+        room_id = kwargs.get('room_id', None)
```

Comment:
`kwargs.get()` method will always return None when no value is provided for the key, so you don't have to include None.